src/components/Button.tsx

```diff
@@ -21,12 +21,20 @@ const Button: React.FC<ButtonProps> = ({
   disabled,
   onClick,
 }) => {
+  const handleClick = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    if (onClick) {
+      onClick();
+      event.currentTarget.blur();
```

Comment:
why? /:

Reply:
ovime ubijas accesibility. kada se nesto klikne, ono treba da bude u fokusu. Ako je problem outline, onda se to resava na druge nacine

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#focus_vs_focus-visible